### ISSUE TITLE
Ashampoo Office

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#994]
**Reported by lallermacher on 2010-10-23 16:16**
It would be nice, if NVDA could support this avesome Programm, ashampoo Office 2010.

**Comment by leonardder** [comment]
I'm not sure what to do with this. Feels a bit too harsh to close, but I wonder whether it is likely that NV Access will ever invest resources into it. @JCSTeh, thoughts?

**Comment by jcsteh** [comment]
We won't invest resources into it, but that doesn't mean someone else can't look into it. P4.

**Comment by Adriani90** [comment]
It seems most parts of this software are accessible with NVDA. For the parts which are not accessible, please create individual issues with exact steps to reproduce. I am closing this for now because it will be unlikely that a developer will search for every issue in ashampoo.
